tools/torch_quant/tests/test_bias_releated_situations.py

```diff
@@ -23,7 +23,6 @@ def test_bias_exist(self):
         dummy = torch.randn(1, 3)
         quantizer = Quantizer(backend=Backend.DISC)
         calib_model = quantizer.calib(model)
-        self.assertTrue(isinstance(model.linear.bias_ob, BiasObserver))
```

Comment:
For disc backend, `bias_ob` should be checked in this ut. Add a standalone or expand ut about the fbgemm or reference backends.

Reply:
For disc backend, `bias_ob` is also not inserted in `calib` stage, which is updated in this pr.